src/components/movieInfo/MovieInfo.tsx

```diff
@@ -0,0 +1,74 @@
+import {useMemo} from 'react';
+import {Text, View} from 'react-native';
+
+import Icon from 'react-native-vector-icons/MaterialIcons';
+import {styles} from './styles';
+
+const MovieInfo = ({
+  movieDetails,
+  genreArr,
+}: {
+  movieDetails: MovieCardData;
+  genreArr: GenreArr;
+}) => {
+  const {
+    title,
+    overview,
+    release_date,
+    vote_average,
+    vote_count,
+    adult,
+    genre_ids,
+  } = movieDetails;
+
+  const {filledStars, borderStars} = useMemo(() => {
+    const roundedRating = Math.round(vote_average / 2);
+    return {
+      filledStars: roundedRating,
+      borderStars: 5 - roundedRating,
+    };
+  }, [vote_average]);
+
+  const movieGenre = genre_ids?.map(genreId => {
```

Comment:
use memo here

If I add comment for one make sure to check at other places for same mistake